web/client/epics/map.js

```diff
@@ -214,11 +222,55 @@ const checkMapPermissions = (action$, {getState = () => {} }) =>
             return loadMapInfo(ConfigUtils.getConfigProp("geoStoreUrl") + "extjs/resource/" + mapId, mapId);
         });
 
+const compareMapChanges = (action$, { getState = () => {} }) =>
+    action$
+        .ofType(CHECK_MAP_CHANGES)
+        .switchMap(({ action, source }) => {
+            const allowedRoles = ['ADMIN', 'USER'];
+            const mapId = getState().map && getState().map.present.mapId;
+            const { currentPage } = getState().feedbackMask;
+            const { user } = getState().security;
+
+            if ((currentPage) !== 'viewer' || allowedRoles.indexOf(user.role) === -1) {
+                return action ? Rx.Observable.of(action) : Rx.Observable.empty();
+            }
+
+            if (mapId) {
+                const url = ConfigUtils.getConfigUrl({ mapId }).configUrl;
+                return Rx.Observable.defer(() => axios.get(url))
```

Comment:
would be possible to take originalMap when doing the first request to get the map info ?

Reply:
Yes, i've done it now :)